.circleci/config.yml

```diff
@@ -22,7 +22,7 @@ e2eTest: &e2eTest
     - run:
         name: Get helm binary
         command: |
-          curl -L https://get.helm.sh/helm-v2.14.3-linux-amd64.tar.gz | tar xvz --strip-components 1 linux-amd64/helm
+          curl -L https://get.helm.sh/helm-v2.16.0-linux-amd64.tar.gz | tar xvz --strip-components 1 linux-amd64/helm
```

Comment:
```suggestion
          curl -L https://get.helm.sh/helm-v2.16.1-linux-amd64.tar.gz | tar xvz --strip-components 1 linux-amd64/helm
```

Reply:
@sslavic This is for the tiller upgrade test. I needed to bump the version because 2.14.3 won't install in KIND as its K8s 1.16 now.

Reply:
Ah, clear, thanks for heads up. 2.16.0 had some bug, so they quickly came out with 2.16.1, hopefully it won't affect these tests.

Reply:
We call `EnsureTillerInstalled` in the tests so tiller is upgraded to 2.16.1 before the tests run. 

e.g. https://github.com/giantswarm/helmclient/blob/8d2f16a18b92557fea612dcae25da5c317739b51/integration/test/basic/basic_test.go#L26

We should maybe only install tiller manually for the upgrade tiller test. But I don't want to invest too much time here. As this goes away with the upgrade to Helm 3.